packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +778,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // update the numbers of lines to be printed from options -<num>|+<num> from user args
+  updatePrintoutLines(userArgs, currentLines) {
+    let beforeLines = currentLines[0], afterLines = currentLines[1];
+    for (let argument of userArgs) {
```

Comment:
This `let` (and the following one) can both be a `const`, btw.  You should generally use `const` when possible.  This area of the code hasn't been touched in a while so it has a lot of `let`s that don't need to be, but when writing new code you should try to use `const` when possible.